README.md

```diff
@@ -30,10 +30,10 @@ You can join these groups and chats to discuss and ask Akka related questions:
 
 In addition to that, you may enjoy following:
 
-- The [news](http://akka.io/news) section of the page, which is updated whenever a new version is released
-- The [Akka Team Blog](http://blog.akka.io)
+- The [news](https://akka.io/news) section of the page, which is updated whenever a new version is released
+- The [Akka Team Blog](https://blog.akka.io)
```

Comment:
`https://akka.io/news` and `https://blog.akka.io` redirect to the same page `https://akka.io/blog/`

Reply:
News: https://akka.io/blog/news-archive.html
Blog: https://akka.io/blog/article-archive.html